AddQtAndroidApk.cmake

```diff
@@ -190,15 +269,53 @@ macro(add_qt_android_apk TARGET SOURCE_TARGET)
         set(TARGET_LEVEL_OPTIONS --android-platform android-${ANDROID_PLATFORM_LEVEL})
     endif()
 
+    if(QT_ANDROID_MIX_SOURCE_DEPLOY)
+        set(QT_ANDROID_MIX_SOURCE_DEPLOY_COMMANDS
+
+        COMMAND ${CMAKE_COMMAND} -E remove_directory ${QT_ANDROID_APP_PACKAGE_SOURCE_ROOT} # Remove every file in our dependencies
+        COMMAND ${CMAKE_COMMAND} -E make_directory ${QT_ANDROID_APP_PACKAGE_SOURCE_ROOT} # Create the directory
+
+        COMMAND ${CMAKE_COMMAND} -E copy ${QT_ANDROID_MANIFEST_SOURCE_IN}/AndroidManifest.xml ${QT_ANDROID_APP_PACKAGE_SOURCE_ROOT}/AndroidManifest.xml
+        COMMAND ${CMAKE_COMMAND} -E copy_directory ${QT_ANDROID_APP_PACKAGE_SOURCE_IN} ${QT_ANDROID_APP_PACKAGE_SOURCE_ROOT} #will erase the first AndroidManifest if another one exist
+            )
+    endif() # QT_ANDROID_MIX_SOURCE_DEPLOY
+
+    if(${CMAKE_BUILD_TYPE} STREQUAL "Release" OR
```

Comment:
This triggers an error when `CMAKE_BUILD_TYPE` is empty (which is the default), since it makes the IF statement ill-formed. We should either force a suitable default value, or (better) check if it's empty before using it.

Reply:
Yes you are right, i never tried with empty CMAKE_BUILD_TYPE. I think we should just check if CMAKE_BUILD_TYPE is empty and display the warning that is already there. Like that qtandroiddeploy will choose by it's own like it was before. But user will get notified.
What are default qt library used when CMAKE_BUILD_TYPE is empty?

Reply:
> What are default qt library used when CMAKE_BUILD_TYPE is empty?

Seems like it is the release build.

Reply:
Fixed in d70068dabe4d1f97ecae4a8ea84ee30b945ebd45